person_capture/gui_app.py

```
@@ -6533,22 +6673,30 @@ def _add_lp_opt(opts: list[str], supported: set[str], name: str, value: str) ->
+        """
         if seek_sec is None:
-            return [], []
+            return [], ""
         try:
             seek_f = max(0.0, float(seek_sec))
         except Exception:
-            return [], []
+            return [], ""
         try:
             preroll_sec = max(0.0, float(os.getenv("PC_HDR_EXPORT_PREROLL_SEC", "2.0")))
         except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)